requirements.txt

```diff
@@ -18,3 +18,5 @@ websocket-client>=0.32.0,!=0.40.0,!=0.41.*,!=0.42.* # LGPLv2+
 # Pin to older version, newer version has issues
 JPype1==0.7.1
 shelljob==0.5.6
+popen
+subprocess.run
```

Comment:
Are these needed here? subprocess and popen (which is part of subprocess) should be part of the standard libraries.

Reply:
thanks. was testing fixes and forgot to rm.